src/List.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import {filterType} from './Constants';
+
+export default function List(props) {
+    const filterDataByList = props.data.filter(item => item.list === props.currList);
```

Comment:
I think you'll find using a subcollection of tasks for each list will make lab 5 easier.